gcp_variant_transforms/transforms/filter_variants.py

```diff
@@ -60,4 +60,7 @@ def _apply_filters(self, variant):
       yield variant
 
   def expand(self, pcoll):
+    # TODO(bashir): It seems there is a type check violation here as the
```

Comment:
This actually gets converted to a [CallableWrapperDoFn](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/core.py#L394). This style is sometimes used if the function is trivial and doesn't have any context. Python magic :)

Reply:
I see, apparently IntelliJ code inspectors did not realize this magic :-)
Looking closer at this, I now see that [the constructor of the base class PTransformWithSideInputs](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/ptransform.py#L520) explicitly does this conversion. I think the problem is the [docstring of the ParDo constructor](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/core.py#L763) which says it expects a DoFn (which is not necessarily true).

Reply:
Please remove the TODO?